### ISSUE TITLE
Slight mask optimization

### ISSUE THREAD

**Issue by kgrabs** [Beatrice-Raws/VapourSynth-insaneAA#1]
For [L134](https://github.com/Beatrice-Raws/VapourSynth-insaneAA/blob/master/insaneAA.py#L134) it's a little faster to use the built-in parameter `scale=2`. The results are not exact but are ±1 with integer formats so that can be chalked up to rounding errors.

A quick test with ffms2 video input gave me

- 264fps vs 285fps in 8 bit
- 211fps vs 241fps in 16 bit
- 150fps vs 191fps in 32 bit

**Comment by DJATOM** [comment]
You can put external whateveryouwant mask clip and it will have zero overhead against internal one. 
The bottleneck of this script is not a mask anyway, but eedi3.

**Comment by kgrabs** [comment]
Yeah that's more or less why I didn't bother to make it a PR, but it wouldn't hurt to add it in next time there's a meaningful update.

**Comment by DJATOM** [comment]
I'll probably break things once again and allow to pass kwargs into filters in the next updates.
